### ISSUE TITLE
App not available in Nextcloud-App-Store

### ISSUE THREAD

**Issue by larewo** [ksainc/data#1]
Hi :-) Your app looks very interesting! Unfortunately I couldn't find it in app store as you described? Maybe I looked at the wrong place? Simon

**Comment by ksainc** [comment]
Hey @larewo,

Yes, the app is not available yet... I need to do some final testing, before its is posted to the app store. Unfortunately I have been busy working on a couple other projects for Nextcloud, mainly the EWS Connector and JMAP Connector. 

I will try to get this in to the store soon, I'll keep the ticket open and inform you when it get posted.

How are you planning on using the app? SIP phones? or something else?

**Comment by larewo** [comment]
Hey ksainc,

thank you very much for your quick reply! 

Yes, I'm planning to access the Nextcloud contacts with my SIP phones. Your app would exactly be what I was looking for. The solutions I found except this weren't really charming for me :-)

I would be happy to hear from you! Thanks in advance for your work!

**Comment by ksainc** [comment]
I agree, there was no good solutions for this that I could find, that is why I built the app!

What brand of SIP phones do you want to use?

**Comment by larewo** [comment]
I'll use Yealink phones...

28.01.2024 14:03:37 Sebastian ***@***.***>:

**Comment by ksainc** [comment]
Morning,

Nice, I've already created templates and tested Yealink as I use them also, give me a couple weeks and I'll have this up in the app store.

Sebastian

**Comment by vzsze** [comment]
I would be interested in this app, too.

**Comment by maxschulze** [comment]
Me too!
